__tests__/utils/fhir.test.tsx

```diff
@@ -0,0 +1,74 @@
+import { getDataRequirementFiltersString } from '../../util/fhir';
```

Comment:
Thanks for adding tests for this file, I forgot to do it in my PR when the file was initially created. That being said, this should just be a `fhir.test.ts` file, not `tsx`, since it ain't a react component

Reply:
good catch